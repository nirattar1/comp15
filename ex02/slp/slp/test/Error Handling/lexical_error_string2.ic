class Stam {
	int run() {
		string a = "this string is english only";

		string b = "this string is also in the עברית language";

		string shimonRules;
	}
}