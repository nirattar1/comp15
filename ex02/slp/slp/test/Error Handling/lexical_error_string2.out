5: Lexical error: illegal character '"'